common/src/main/java/org/samo_lego/taterzens/mixin/network/ServerGamePacketListenerImplMixin_PacketFaker.java

```diff
@@ -91,7 +90,7 @@ private void changeEntityType(Packet<?> packet, GenericFutureListener<? extends
             ClientboundPlayerInfoPacket playerAddPacket = new ClientboundPlayerInfoPacket(ADD_PLAYER);
             //noinspection ConstantConditions
             ((ClientboundPlayerInfoPacketAccessor) playerAddPacket).setEntries(
-                    Arrays.asList(new ClientboundPlayerInfoPacket.PlayerUpdate(profile, 0, GameType.SURVIVAL, npc.getTabListName()))
+                List.of(new ClientboundPlayerInfoPacket.PlayerUpdate(profile, 0, GameType.SURVIVAL, npc.getTabListName()))
```

Comment:
Please don't use `List.of` here, as it will result in an immutable collection, making it crash with e.g. vanish plugins which expect default (mutable) one ;).

Reply:
Will do, yep. Are you sure `Arrays.asList` doesn't cause problems - calling `.remove` on that will also throw an exception.

Reply:
I know that `List.of` has conflicted previously, whereas `Arrays.asList` was fine. Probably sth other than mutability then, I don't exactly remember.